docs/contributing/intro.md

```diff
@@ -0,0 +1,73 @@
+# Contribute to stravalib
```

Comment:
this is just a stub page so we can have a new contributing section with it's own navigation for easier discovery.